web/styles/settings.css

```diff
@@ -1308,20 +1308,22 @@ label.preferences-radio-choice-label {
         .tab-container {
             box-sizing: border-box;
             height: $settings_header_height;
-            padding: 6px;
+            display: flex;
+            align-items: center;
+            justify-content: center;
             background-color: var(--color-background-modal-bar);
             border-bottom: 1px solid var(--color-border-modal-bar);
 
             @media (width >= $md_min) {
                 .tab-switcher {
                     position: absolute;
                     display: flex;
-                    left: 10px;
+                    left: 0.7142em; /* 10px at 14px em */
```

Comment:
Is the layout going to tolerate being shifted left by an em value? My feeling is that we should tread carefully on horizontal matters (padding, width, left/right offsets, etc.)

Reply:
This left margin and absolute positioning have been removed -- it's now in flexbox and centers itself horizontally in the left width space

Reply:
I'll just avoid changing it in this commit, since it gets removed later.